airflow/configuration.py

```diff
@@ -38,7 +38,7 @@
 
 # show Airflow's deprecation warnings
 warnings.filterwarnings(
-    action='default', category=DeprecationWarning, module='airflow')
+    action='always', category=DeprecationWarning, module='airflow')
```

Comment:
Why this change? And what effect does this change have?

Reply:
This was for testing and accidentally added to the commit. I have changed it back. Sorry for the confusion.